src/hubAuth.ts

```diff
@@ -9,12 +9,12 @@ import * as os from 'os';
 import * as shell from 'shelljs';
 import { debug } from 'debug';
 
-import { fs } from '@salesforce/core';
+import { AuthFields, fs } from '@salesforce/core';
 import { env } from '@salesforce/kit';
 
 // this seems to be a known eslint error for enums
 // eslint-disable-next-line no-shadow
-enum AuthStrategy {
+export enum AuthStrategy {
```

Comment:
did you mean to have a `NONE` option like I saw in the NUT?

Reply:
or any non-matching value would be fine?

Reply:
@mshanemc there is a `NONE` option: https://github.com/salesforcecli/cli-plugins-testkit/pull/32/files#diff-3ee7b974ef4865c77089d3727bf236372881bf206a13035bbd269e1b2e3d5305L21